scripts/decode-eth-eip1559.js

```diff
@@ -0,0 +1,13 @@
+const { Transaction: TransactionFactory, FeeMarketEIP1559Transaction } = require('@ethereumjs/tx');
+const ethUtil = require('ethereumjs-util');
+const prompt = require('prompt-sync')();
+
+main();
+
+async function main() {
+
+	const data = '0x' + prompt('Enter the transaction hex from your recovery JSON: ');
```

Comment:
Should probably use process.stdin there's no reason to pull in a library for something so trivial. Read the Node.js docs for handling IO it has similar APIs to C++ here for dealing with IO so this should be straightforward enough for you.